### ISSUE TITLE
[FEATURE] New World Cup format 

### ISSUE THREAD

**Issue by wsbrenk** [akasolace/HO#833]
Hattrick announced new NT formats (https://www.hattrick.org/Help/Rules/NewNationalTeams.aspx).

We have to adapt HO's NT support to those changes. A lot of chpp files would be affected. Further Infos see here https://www.hattrick.org/Forum/Read.aspx?n=1&nm=40&t=17380663&v=0

**Comment by akasolace** [comment]
@wsbrenk what do you think we need to do here?  We don't really do anything related to world cup in HO!, no ?

**Comment by wsbrenk** [comment]
to be honest - i have no idea if there are any users working with HO an national teams and what they expect concerning this new format. and i think this kind of matches are ignored by HO.

**Comment by wsbrenk** [comment]
I am convinced that it could have a huge advertising impact if we could work together with some national team coaches.
perhaps we are surpassing the youthclub in the annual chpp applications ranking;-)

**Comment by akasolace** [comment]
yeah I am just not sure what we can offer really. But the way I understand it, it is potentially more a new feature than a bug fixe so maybe the ticket is for another release and not for 5.0, right ?

**Comment by wsbrenk** [comment]
@akasolace confirmative
